index.js

```diff
@@ -67,6 +81,31 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			var progressEnabled = progress.emitter.listeners('progress').length > 0;
+			if (progressEnabled) {
```

Comment:
I didn't thought node internals was smart enough, and tried to optimize this.
But now I don't think it is worth anymore.